doc/api/date/date-to-parts-formatter.md

```diff
@@ -0,0 +1,190 @@
+## .dateToPartsFormatter( [options] ) ➜ function( value )
+
+Return a function that formats a date into parts tokens according to the given
+`options`. The default formatting is numeric year, month, and day (i.e., `{
+skeleton: "yMd" }`.
+
+The returned function is invoked with one argument: the Date instance `value` to
+be formatted.
+
+### Parameters
+
+**options**
+
+Please, see [.dateFormatter() options](./date-formatter.md#parameters).
+
+**value**
+
+Date instance to be formatted, eg. `new Date()`;
+
+### Returns
+
+An Array of objects containing the formatted date in parts. The returned structure looks like this:
+
+```js
+[
+  { type: "day", value: "17" },
+  { type: "weekday", value "Monday" }
+]
+```
+
+Possible types are the following:
+
+- `day`
+
+  The string used for the day, e.g., `"17"`, `"١٦"`.
```

Comment:
Yeap, I think it needs overall improvement. I dislike the current format of the API, I am wondering if we had something like https://github.com/rxaviers/relative-time#formatdate-options instead.